JSQMessagesViewController/Layout/JSQMessagesCollectionViewFlowLayout.m

```diff
@@ -541,10 +541,17 @@ - (CGSize)jsq_avatarSizeForIndexPath:(NSIndexPath *)indexPath
 
 - (UIAttachmentBehavior *)jsq_springBehaviorWithLayoutAttributesItem:(UICollectionViewLayoutAttributes *)item
 {
-    UIAttachmentBehavior *springBehavior = [[UIAttachmentBehavior alloc] initWithItem:item attachedToAnchor:item.center];
-    springBehavior.length = 1.0f;
-    springBehavior.damping = 1.0f;
-    springBehavior.frequency = 1.0f;
+    UIAttachmentBehavior *springBehavior = nil;
```

Comment:
for better readability and a smaller diff, let's return early here instead:

````objective-c
- (UIAttachmentBehavior *)jsq_springBehaviorWithLayoutAttributesItem:(UICollectionViewLayoutAttributes *)item 
{
    if (CGSizeEqualToSize(item.frame.size, CGSizeZero)) {
        // adding a spring behavior with zero size will fail in in -prepareForCollectionViewUpdates:
        return nil;
    }
    // rest of original method
}
````